src/app/result/[role]/page.tsx

```diff
@@ -54,8 +55,24 @@ const Results: React.FC = async () => {
 
 export const ShowRolesWrapper = async ({ path }: { path: string }) => {
   const availableRoles = await generateRolesWithHref(path)();
-
-  return <SelectRoleResults roles={availableRoles} />;
+  const def = {
+    id: "",
+    href: path,
+    label: "No role",
+    current: false,
+    completed: false,
+    started: false,
+    currentCompleted: false
+  };
+  availableRoles.unshift(def as Section);
+  if (path.includes("result")) {
+    return (
+      <SelectRoleResults roles={availableRoles} />
+    )
+  }
+  return (
+      <ShowTechSearchWrapper roles={availableRoles} />
+  );
```

Comment:
Not sure I'm following here, what does this "result" check do exactly?

Reply:
yes, both the search the expert page and the anonymized results page used SelectRoleResults. I have now created the ShowTechSearchWrapper to extend the search capabilities for 'search the expert' page. With checking on "result" we can still show the old version for the anonymized results page

Reply:
Okay that's clear!